Scripts/Core/Contexts/EditingContextManager.cs

```diff
@@ -47,13 +47,14 @@ internal static IEditingContext defaultContext
                 var availableContexts = GetAvailableEditingContexts();
                 var context = availableContexts.Find(c => c.Equals(s_DefaultContext)) ?? availableContexts.First();
 
-                var defaultContextName = settings.defaultContextName;
-                if (!string.IsNullOrEmpty(defaultContextName))
-                {
-                    var foundContext = availableContexts.Find(c => c.name == defaultContextName);
-                    if (foundContext != null)
-                        context = foundContext;
-                }
+                // TODO: Fix context switching while EXR is running
```

Comment:
How come? It doesn't always seem to screw up.

Reply:
In my testing, the rays would break every time. I see a big orange quad instead of the ray, and the cone disappears. I figured this was a separate concern from introducing minimal mode, so I didn't want to hold up the PR on fixing this.

Reply:
Aaaand now it's not breaking 🤔 
I will revert

Reply:
On further investigation, it seems only to break when going from EditorXR -> Minimal mode. I'll investigate this a bit more

Reply:
Yeah, wasn't able to get to the bottom of this. We could force a whole restart of the VRView as a fix, but if you get to the root cause of it kudos to you.

Reply:
Actually, it doesn't seem to matter in what order I switch. It appears to happen every time I switch once the HMD and controllers are recognized. I did have a few successful switches earlier that I can't account for.

It appears that at least the controller issue is due to the fact that after switching the camera parent has a scale of 1000. I'm trying to find out where this happens.

Reply:
I think I've figured it out. It's not that switching causes the viewer scale to change, but if you switch with a non-zero viewer scale, the proxy ray shows up wrong. I'll see if I can find a fix.